content/en/docs/05.0/_index.en.md

```diff
@@ -5,7 +5,7 @@ weight: 5
 
 Previously in the lab...
 
-> Question: Do we need two running mariadb containers at the same time for this lab?
+> Question: Do we need two running Maria containers at the same time for this lab?
```

Comment:
Below (one of the next diffs) you write MariaDB and not Maria

Reply:
Fixed